functions/concepts/scopes/app.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_concepts_filesystem]
+require "functions_framework"
+
+def heavy_computation
+  Time.now
+end
+
+def light_computation
+  Time.now
+end
+
```

Comment:
Need START/END sample tags for `functions_concepts_scopes`. Based on the [python samples](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/functions/concepts/main.py) the tagged region should surround this call to `heavy_computation` and the function below.

Also, add some comments emphasizing that this call is establishing a global (instance-wide) result and runs at cold-start. (Again, see the python samples.)